ext/ftp/ftp.c

```diff
@@ -1145,7 +1147,8 @@ ftp_size(ftpbuf_t *ftp, const char *path, const size_t path_len)
 	if (!ftp_getresp(ftp) || ftp->resp != 213) {
 		return -1;
 	}
-	return atol(ftp->inbuf);
+	ZEND_ATOL(res, ftp->inbuf);
```

Comment:
Gotta wonder, why the heck doe ZEND_ATOL include a variable assignment instead of just returning the value?

Reply:
Looks like an unfortunate design decision, but probably too much BC break if we change that.